rust/processor/src/db/postgres/models/fungible_asset_models/v2_coin_to_fungible_asset_mappings.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use crate::{
+    db::common::models::fungible_asset_models::raw_v2_coin_to_fungible_asset_mappings::{
+        CoinToFungibleAssetMappingConvertible, RawCoinToFungibleAssetMapping,
+    },
+    schema::coin_to_fungible_asset_mappings,
+};
+use field_count::FieldCount;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, FieldCount, Identifiable, Insertable, Serialize)]
+#[diesel(primary_key(coin_type))]
+#[diesel(table_name = coin_to_fungible_asset_mappings)]
+pub struct CoinToFungibleAssetMapping {
+    pub coin_type: String,
+    pub fungible_asset_metadata_address: String,
+    pub last_transaction_version: i64,
+}
+
+impl CoinToFungibleAssetMappingConvertible for CoinToFungibleAssetMapping {
+    fn from_raw(raw: RawCoinToFungibleAssetMapping) -> Self {
+        Self {
+            coin_type: raw.coin_type,
+            fungible_asset_metadata_address: raw.fungible_asset_metadata_address,
+            last_transaction_version: raw.last_transaction_version,
+        }
+    }
+}
```

Comment:
Instead of having a custom trait, u could just `impl From<RawCoinToFungibleAssetMapping> for CoinToFungibleAssetMapping` and call `.into()` when needed

Reply:
nvm seems like all the other models have a trait like this lol... dunno why but ok

Reply:
yup.